### DISCUSSION TITLE
Windows size

### DISCUSSION THREAD

**Discussion by gigli69** [BobanSpasic/PSS-Revive#8] [Q&A]
On my laptop with screen of 1366*768 I cannot see the entire desk: I can only modify the windows width, but not the windows height,

Could you arrange this things?
Thanks

**Comment by BobanSpasic** [comment]
Hi.
Which operating system? Windows or Linux?

**Reply by gigli69** [reply]
Under Linux, but It's the same under Win : the window can only be resized by width, not by height, so I can't see all, as you can see on this screenshot:

![PSS-revive](https://github.com/BobanSpasic/PSS-Revive/assets/71589194/59d3d9a0-4c5a-4972-995c-d39f4b0bdcc4)

**Reply by BobanSpasic** [reply]
I did some tests now. The simplest workaround is to add scrollbars.
A better solution would be to adjust it over PPI of the form, but that would be playing in with the unknown for me. I did some tests now, and the form resizes nicely if I change the displaying PixelPerInch, except for the Logo.
I'll try to do something about it.

btw. moving this from Discussion to Issues

**Comment by eugene-s-nesdev** [comment]
Same issue on 1024x600 eeePC laptop:
![big](https://github.com/BobanSpasic/PSS-Revive/assets/13199529/599b38da-e3fe-4318-88c9-ab6aa4296552)

**Reply by BobanSpasic** [reply]
@eugene-s-nesdev is that on Windows? My main development platform is on Windows, and it is easier for me to compile test builds for Windows. If it is Windows, could you try the version from the attachment
[PSS-Revive.zip](https://github.com/BobanSpasic/PSS-Revive/files/13854313/PSS-Revive.zip)

**Comment by eugene-s-nesdev** [comment]
Thank you. Still big on 1366x768, but now it's have scroll bars:
![02](https://github.com/BobanSpasic/PSS-Revive/assets/13199529/1c8ba063-9c7c-448e-ac8c-bf33404e45e3)
![01](https://github.com/BobanSpasic/PSS-Revive/assets/13199529/6266e54b-0836-4a04-9ffe-67c2a6f9a4eb)

**Reply by BobanSpasic** [reply]
I've spent the afternoon searching for a way to auto-adapt the window to the screen size, but every method I've tried had some troubles (graphic deformations). The most troublesome part are the sliders - I just can't get them to scale to another size.  
At the end, I've decided to go for scroll-bars.
